src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -211,7 +213,10 @@ CreateModifyPlan(Query *originalQuery, Query *query,
 		return multiPlan;
 	}
 
-	ereport(DEBUG2, (errmsg("Creating router plan")));
+	if (*multiShardModifyQuery == false)
+	{
+		ereport(DEBUG2, (errmsg("Creating router plan")));
```

Comment:
Why don't we want to show this debug message? Explain will still show this query as using `Citus Router`. This complicates the signature of `RouterJob`, so not sure if it's worthwhile.